src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/judgeapproval/JudgeApprovalResolver.java

```diff
@@ -103,7 +103,7 @@ void handleApprovable(Approvable approvable, JudgeApproval judgeApproval, String
                     approvable.replaceDocument(judgeApproval.getAmendedDocument());
                 }
                 approvable.setOrderStatus(APPROVED_BY_JUDGE);
-                approvable.setApprovalDate(LocalDateTime.now());
+                approvable.setApprovalDate(judgeApproval.getOrderApprovedDate().atStartOfDay());
```

Comment:
I think orderApprovedDate must be a mandatory field in CCD definition. 
Or shall we add a null check and default to current date?